EXILED/Exiled.Events/EventArgs/Scp079/RecontainingEventArgs.cs

```diff
@@ -0,0 +1,41 @@
+// -----------------------------------------------------------------------
+// <copyright file="RecontainingEventArgs.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.EventArgs.Scp079
+{
+    using Exiled.API.Features;
+    using Exiled.Events.EventArgs.Interfaces;
+
+    /// <summary>
+    /// Contains information before SCP-079 gets recontained.
+    /// </summary>
+    public class RecontainingEventArgs : IDeniableEvent
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RecontainingEventArgs" /> class.
+        /// </summary>
+        /// <param name="recontainer">The <see cref="BreakableWindow"/> instance.</param>
+        public RecontainingEventArgs(BreakableWindow recontainer)
+        {
+            Recontainer = Player.Get(recontainer?.LastAttacker.Hub);
```

Comment:
recontainer can't be null but LastAttacker could be also please make IsAutomatic with recontainer.LastAttacker.IsSet

Reply:
LastAttacker.Hub can't be null.
Anyways if recontainer.LastAttacker is null Player.Get() will return a null so it's the same thing